### PR TITLE
adding_users.rst: We create users with ansible now

### PR DESCRIPTION
**Pull request by dmick**
Signed-off-by: Dan Mick dan.mick@redhat.com

### CODE REVIEW

File: doc/adding_users.rst [hunk 1]
```diff
@@ -74,15 +74,10 @@ Adding users (lab admins)
...
-     mkdir .ssh
-     cat > .ssh/authorized_keys
-     <ssh key>
-     ^D
-     chown -R --reference=. .ssh
-     chmod 700 .ssh
-     chmod 600 .ssh/*
+#. Run ansible to create the user::
 
+     cd <path-to-ceph-cm-ansible>
```
Comment (andrewschoen) [inline doc/adding_users.rst:79]: We should probably mention the step of adding the user to the secrets repo here somewhere.

**Comment by andrewschoen** [comment]
Linking to the usage documentation for the users role might be helpful as well.

https://github.com/ceph/ceph-cm-ansible/tree/master/roles/users#usage

File: doc/adding_users.rst (hunk 1 above)
Comment (dmick) [inline doc/adding_users.rst:79]: See lines 73-75 (unchanged)

Comment (andrewschoen) [inline doc/adding_users.rst:79]: Ok, thanks, I see that now.  I need to add some docs somewhere (most likely ceph-sepia-secrets) on how `lab_users` and `admin_users` are used to provide the `managed_users` and `managed_admin_users` vars for the users role.

**Comment by dmick** [comment]
Forcepushed to add url (and to pull commentary out of the 'command' textbox)

**Pull request merged**